src/SFA.DAS.ApplyService.Web/Services/TrusteeExemptionService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using SFA.DAS.ApplyService.Configuration;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.Web.Services;
+
+public class TrusteeExemptionService : ITrusteeExemptionService
+{
+    private readonly IConfigurationService _configurationService;
+
+    public TrusteeExemptionService(IConfigurationService configurationService)
+    {
+        _configurationService = configurationService;
+    }
+
+    public async Task<bool> IsProviderExempted(string ukprn)
+    {
+        var config = await _configurationService.GetConfig();
+        var exemptionListing = config.ProvidersExemptedFromHavingTrustees;
+        var exemptions = exemptionListing.Split(',');
+
+        var isPresent = exemptions.Any(x => x == ukprn);
+        return isPresent;
+    }
+}
+
+public interface ITrusteeExemptionService
```

Comment:
Can the interface be moved on the top of the file or to its own file?

Reply:
moved to its own file